layouts/partials/features.html

```diff
@@ -13,13 +13,13 @@
                         {{ with $element.url }}
                         <a href="{{ $element.url }}">
                         {{ end }}
-                            <div class="icon">
+                            <div class="icon-outlined">
                                 <i class="{{ .icon }}"></i>
                             </div>
                         {{ with $element.url }}
                         </a>
                         {{ end }}
-                        <h3>{{ $element.name | markdownify }}</h3>
+                        <h3 class="h4">{{ $element.name | markdownify }}</h3>
```

Comment:
sounds weird h3 and class h4.

Reply:
Yeah. I just ripped this off from the Boostrapious live demo/raw distro. We download the raw distro from them and pay for a non-attribution license.

Reply:
I leave the comment open to remember on the final check